### PR TITLE
PD-296: Add a note to CLI topics referring users to Kubernetes topics

### PR DESCRIPTION
**Pull request by andreipope**
Signed-off-by: Andrei Popescu <andrei.popescu@gmail.com>

### CODE REVIEW

**Comment by andreipope** [comment]
@nathan-px Hey Nathan, please review this PR. Thanks!

File: content/reference/CLI/basics.md [hunk 1]
```diff
@@ -62,23 +62,23 @@ Use "pxctl [command] --help" for more information about a command.
 As seen above, `pxctl` provides the capability to perform fine-grained control of the PX resources cluster-wide. Also, it lets the user manage volumes, snapshots, cluster resources, hosts in the cluster and software upgrade in the cluster.
 {{</info>}}
 
-In addition, every command takes in a `--json` which converts the output to machine parsable `JSON` format. You can do something like the following to save the the output in `JSON` format:
+In addition, every command takes in a `--json` which converts the output to a machine parsable `JSON` format. You can do something like the following to save the output in `JSON` format:
```
Comment (nathan-px) [inline content/reference/CLI/basics.md]:
```suggestion
In addition, every command takes in a `--json` flag which converts the output to a machine-parsable `JSON` format. You can do something like the following to save the output in `JSON` format:
```

File: content/reference/CLI/updating-volumes.md [hunk 2]
```diff
@@ -280,6 +284,10 @@ Volume	:  485002114762355071
 			Node 	 :  172.31.55.104
 ```
 
+### Related topics
+
+For more information about how to dynamically resize a volume (PVC) using Kubernetes and Portworx, refer to the [Resize a Portworx PVC](/portworx-install-with-kubernetes/storage-operations/create-pvcs/resize-pvc/) page.
```
Comment (nathan-px) [inline content/reference/CLI/updating-volumes.md]: These all look mostly good to me. I think you can drop the "how to" and switch to the participle verb form in them to make it more readable:
```suggestion
For more information about dynamically resizing a volume (PVC) using Kubernetes and Portworx, refer to the [Resize a Portworx PVC](/portworx-install-with-kubernetes/storage-operations/create-pvcs/resize-pvc/) page.
```

File: content/reference/CLI/updating-volumes.md [hunk 3]
```diff
@@ -121,6 +121,10 @@ Volume	:  970758537931791410
 As shown above, the `shared` field is set to `yes` indicating that `clitest` is now a shared volume
 
 
+### Related topics
+
+* For more information about how to create shared Portworx volumes through Kubernetes, refer to the [Create shared PVCs](/portworx-install-with-kubernetes/storage-operations/create-pvcs/create-shared-pvcs/) page.
```
Comment (nathan-px) [inline content/reference/CLI/updating-volumes.md]:
```suggestion
* For more information about creating shared Portworx volumes through Kubernetes, refer to the [Create shared PVCs](/portworx-install-with-kubernetes/storage-operations/create-pvcs/create-shared-pvcs/) page.
```

Comment (andreipope) [reply content/reference/CLI/updating-volumes.md]: > Approved, but with a small grammar comment.

Nice catch. Thanks!

**Pull request merged**